zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3130,6 +3130,51 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /groups/{groupId}/mapping-rules/search:
```

Comment:
❓ Shouldn't this be `mappings`? I thought we were using that term instead of `mapping-rules` these days.

Reply:
The mapping controller has `mapping-rules` as path, also other endpoints in GroupController have the same path

Reply:
Raised the incosistency [here](https://camunda.slack.com/archives/C06UYJMMETZ/p1746691193965769). Lets see what the outcome is.